frontend/src/fixedComponents/Navbar.js

```diff
@@ -2,20 +2,16 @@ import React from 'react';
 import { connect } from 'react-redux';
 import './fixed.css';
```

Comment:
If you keep all the css in a folder and import them into index.css and import directly from there, you do not need to import them separately for each page.